markdown/chapters/meandeck/brainstorm.md

```diff
@@ -1,21 +1,58 @@
 ---
 authors: angrybacon, Doishy
 order: 1
-title: Brainstorm
+title: Brainstorm in Doomsday
 ---
 
+*Preface: We will be using terminology and abbreviations introduced in the
+Basics chapter of the wiki. If you have not read the Basics chapter it is
+recommended you do so first as this builds upon the concepts introduced there.
+
+Brainstorm is one of the most powerful spells available in the Legacy format at
+this time. There have been numerous articles written on this *seemingly
+innocuous at first glance* one mana cantrip. We'll assume you know the basics of
+casting Brainstorm, and instead focus on the specifics of using it as a part of,
+or to initiate, your doomsday piles. In many cases it can be used as any other
+cantrip, but it has unique applications to Doomsday piles by allowing you to
+swap cards in hand for cards in the pile.
+
 In the basics we alluded to the use of Brainstorm as being a special case when
-discussing the use of cantrips to draw into our Doomsday piles. More than a
-cantrip to start consuming your piles, Brainstorm can effectively draw 3 cards
-at once from said pile unlike other options from your suite of cantrips. This,
-combined with otherwise *dead* cards from hand can turn a resource light turn
-into a winning line.
+discussing the use of cantrips to dig into our Doomsday piles. More than just a
+cantrip to start consuming your piles, Brainstorm can enable you to access three
+cards at once from said pile unlike other options from your suite of cantrips
+which only allow access to one. This, combined with what could be otherwise
+*dead* cards from hand can turn a resource light turn into a winning one.
+
+Brainstorm piles allow you to win with limited initial mana resources. For
+example; if you are constrained on mana against Wasteland or opposing Rishadan
+Port activations you can still combo out with only a Brainstorm post-Doomsday as
+long as you have at least one extra card in hand to put back. Sometimes it can
+be difficult to identify when you have access to a Brainstorm pile, or when your
+Brainstorm is better off simply as the second cantrip in a Double Cantrip pile.
+With this guide, and some practice, you should become confident in this decision
+making in no time.
+
+## Brainstorm Pile - Brainstorm Available
+
+> {B}{B}{B} + Brainstorm + X
```

Comment:
I saw that you explained how the pile notation reads in the "Basics" chapter but I think leaving the U requirement out could be confusing when piles require Brainstorm (or cantrips in general) in hand but aren't part of the initial cost of the pile (Brainstorm pile typically cheats on mana vs. cards in hand pre-Doomsday). Isn't it clearer if the cost on the left lists all costs when they are already determined? ie.

```raw
BBB + U + Brainstorm + Ponder
```

In this case it's clear that you only need to pay for the Brainstorm upfront and Ponder will be paid for with mana from the pile. Whereas just `BBB + Brainstorm + Ponder` might make the reader think Ponder mana is a requirement right next to that of Brainstorm.

Thoughts on removing the BBB part of all costs? I can't remember why I liked it before but while presenting a pile, it should be pretty clear to the reader that this is post-Doomsday and that you need to resolve Doomsday first.

Reply:
Hmm, happy to have either way. I'll leave that up to you.